app/js/rot13.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
 var ROT13Cipher = (function () {
-  // your code here
+  var plain = "abcdefghijklmnopqrstuvwxyz".split('');
+  var cipher = "nopqrstuvwxyzabcdefghijklm".split('');
+  var cipher25 = "zabcdefghijklmnopqrstuvwxy".split('');
+
+  var Encode = function(string){
+    var display = function(){
+      return result();
+    }
+
+    var result = function(){
+      var new_string = '';
+      for(var i = 0; i < string.length; i++){
+        var charToReplace = string.charAt(i);
+        var indexInPlain = plain.indexOf(charToReplace);
```

Comment:
We haven't covered this, but using .indexOf instead of a specific index turns this from an O(n) problem to an O(n^2) problem I think. Otherwise, this is great.